cmd/influxd/run.go

```diff
@@ -225,11 +225,17 @@ func openServer(path string, u *url.URL, b *messaging.Broker, initializing, conf
 			openServerClient(s, joinURLs)
 		}
 	} else if !configExists {
-		// We are spining up an server that has no config,
+		// We are spining up a server that has no config,
 		// but already has an initialized data directory
 		joinURLs = []*url.URL{b.URL()}
 		openServerClient(s, joinURLs)
 	} else {
+		if len(joinURLs) == 0 {
```

Comment:
It wouldn't spin up as it didn't know how to connect to a broker.  Long term this needs to come from some meta data I think, but I'm not exactly sure how we want the behavior.  For now, this fixes it but is a pretty naive approach.